### DISCUSSION TITLE
IBAN empty, but when updating claims in use

### DISCUSSION THREAD

**Discussion by filip-van-hoeckel** [firefly-iii/firefly-iii#9567] [Firefly III] [answered]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem
- [x] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

IBAN for account is empty, however when updating it claims IBAN is in use. However transfers to this account from another personal account are not recognized as internal. Please advise, thanks.

### Debug information

Debug information generated at 2024-12-22 10:16:59 for Firefly III version **v6.1.24**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>6.1.24 / v2.1.0 / 25 (exp. 25)</td></tr><tr><td>PHP version</td><td>8.3.14 (64bits) / fpm-fcgi / Linux x86_64</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>8388608 (8 MB)</td></tr><tr><td>Database drivers</td><td><strong>*mysql*</strong>, </td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Amsterdam + Europe/Amsterdam</td></tr><tr><td>App environment</td><td>local, debug: true</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>notice, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td></td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>never (never)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td><span title="Has rules">:wrench:</span></td></tr><tr><td>Session start</td><td>2024-12-01 00:00:00</td></tr><tr><td>Session end</td><td>2024-12-31 23:59:59</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>nl_NL</td></tr><tr><td>User locale</td><td>nl_NL</td></tr><tr><td>Locale(s) supported</td><td>nl_NL.utf8: :white_check_mark:<br>nl_NL.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 Edg/131.0.0.0</td></tr></tbody></table>

### Expected behaviour

Store and show IBAN. Recognize internal transfers

**Comment by JC5** [comment, marked as answer]
Hi, thanks for opening an issue.

My guess is there is an expense or revenue account with said IBAN. Empty that one, then come back to this account and update it.

**Reply by filip-van-hoeckel** [reply]
Transactions were booked on the IBAN; apparently that blocks assigning the proper IBAN. Thanks.
